wordpress-mobile/WordPress-Android
# Site creation rework: category screen

Fixes #6925 

To test:
* With the app logged in, go to site picker and hit the "+" action to add a new site
* Choose "Create WordPress.com site"
* Notice the "Create New Site" screen come up and asking to select the site category
* Tapping on any of the 3 big cards should bring up a Toast with the category selected
